### ISSUE TITLE
Reflected XSS vulnerability in username field.

### ISSUE THREAD

**Issue by Firewaters** [Microsoft/adfsWebCustomization#34]
The paginated JavaScript could cause a malicious user to do a reflected XSS attack. This was happening before the latest 2018 release centeredUi as well. 

`adfs/ls/?username=rr%22%3E%3Cimg%20src=x%20onerror=prompt(document.domain)%3Errrrrr`

**Comment by bongiovimatthew-microsoft** [comment]
Thanks for bringing this up @Firewaters. We recently merged [a fix](https://github.com/Microsoft/adfsWebCustomization/pull/36) for the issue. The root cause is that we are constructing an "Identity Banner" when we display the password page. That Identity Banner displays the username that was entered, either from the username textbox, or the username field set on the query string. We were previously setting the innerHTML field on the Banner, which meant that it was possible to inject malicious HTML. We have switched to use innerText, which will only set the text content on the node. This will prevent any malicious HTML from being executed when we add the Identity Banner node to the DOM tree.
